src/urllib3/contrib/pyopenssl.py

```diff
@@ -386,8 +386,10 @@ def _drop(self):
         else:
             self._makefile_refs -= 1
 
-
-WrappedSocket.makefile = backport_makefile
+    def makefile(self, mode="r", buffering=None, *args, **kwargs):
+        text = socket_makefile(mode, buffering, *args, **kwargs)
```

Comment:
Shouldn't we pass `self` as the first arg here as well? I can see that you did it for `securetransport.py`